evento/src/setupTests.js

```diff
@@ -6,3 +6,59 @@ global.waitForFetches = () => {
 		}, 50);
 	});
 };
+
+// Create a large random integer to prevent collisions
+const generateId = () => Math.round(Math.random() * Number.MAX_SAFE_INTEGER);
+// Convert number to base36 string to get random combination of alphabets and numbers
+const generateString = (n) => (Math.random().toString(36)+'00000000000000000').slice(2, n+2);
+
+// Factories
+global.generateUser = () => {
```

Comment:
Do we want these to be global? Or for example, "Mock", so that it could be called "Mock.generateUser()" or something? Factory? Mocker? Opinions?